app/controllers/pages_controller.rb

```diff
@@ -63,7 +63,7 @@ def destroy
       return redirect_to @back_url
     end
 
-    unless PageRepository.destroy(page)
+    unless page.destroy
```

Comment:
I think we need to call `destroy_and_update_form` here?

Reply:
Ah good spot, missed that we had that method 🤦

Reply:
Thanks, I've updated and made sure we handle the `ActiveRecord::RecordNotFound` error in the controller and added a test for this.